src/data_gradients/managers/detection_manager.py

```diff
@@ -56,6 +66,8 @@ def __init__(
         :param n_image_channels:        Number of channels for each image in the dataset
         :param remove_plots_after_report:  Delete the plots from the report directory after the report is generated. By default, True
         """
+        assert feature_extractors is None or config_path is None, "`feature_extractors` and `config_path` cannot be specified at the same time"
+
```

Comment:
I prefer to raise this type of error the same way as with `class_names_to_use`, i.e.
``` python
if ...:
    raise RuntimeError(...)
```
But not blocking it's not important

Reply:
fixed